src/components/structures/LeftPanel.tsx

```diff
@@ -309,18 +309,15 @@ export default class LeftPanel extends React.Component<IProps, IState> {
                 }
                 break;
         }
+
         const navAction = getKeyBindingsManager().getNavigationAction(ev);
-        switch (navAction) {
-            case KeyBindingAction.NextLandmark:
-                ev.stopPropagation();
-                ev.preventDefault();
-                LandmarkNavigation.navigateToNextLandmarkFrom(Landmark.ROOM_SEARCH);
-                break;
-            case KeyBindingAction.PreviousLandmark:
-                ev.stopPropagation();
-                ev.preventDefault();
-                LandmarkNavigation.navigateToPreviousLandmarkFrom(Landmark.ROOM_SEARCH);
-                break;
+        if (navAction) {
+            ev.stopPropagation();
+            ev.preventDefault();
+            LandmarkNavigation.findAndFocusNextLandmark(
+                Landmark.ROOM_SEARCH,
+                navAction === KeyBindingAction.PreviousLandmark,
+            );
```

Comment:
I think this should check it is one of the expected 2 keybinding actions, otherwise other navigation bindings would also cause landmarks to move: https://github.com/matrix-org/matrix-react-sdk/blob/c843a4163a4f03dbca2bc78ffb1ed64c247e53b9/src/accessibility/KeyboardShortcuts.ts#L277-L295

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/12190/commits/114cc77acdcc8b9527e139710e162a951896d28b